### PR TITLE
promoting version 7.2.4-7

### PR DESCRIPTION
**Pull request by alon-zada**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1 @@
-.idea
```
Comment (yuvallevy2) [inline .gitignore:1]: revert

File: active_active_database_readme.md [hunk 2]
```diff
@@ -722,7 +722,7 @@ Note - Modules are currently in preview for Active-Active databases.
 To use modules with Active-Active databases enable the alpha feature flag on all participating clusters:  
 To enable modules for Active-Active databases, set a boolean environment variable with the name "ENABLE_ALPHA_FEATURES" to True. This variable can be set via the redis-enterprise-operator pod spec, or through the operator-environment-config Config Map.
 
-Important note:
```
Comment (yuvallevy2) [inline active_active_database_readme.md:725]: revert

File: .gitignore (hunk 1 above)
Comment (alon-zada) [reply .gitignore]: done.

File: active_active_database_readme.md (hunk 2 above)
Comment (alon-zada) [reply active_active_database_readme.md]: done.

**Review by yuvallevy2 — APPROVED** [review]

**Pull request merged**